File: SDKs/Aspose.Cells_Cloud_SDK_For_Ruby/lib/aspose_cells_cloud/models/hyperlink.rb
module AsposeCellsCloud
  # 
  class Hyperlink < BaseObject
    attr_accessor :address, :area, :screen_tip, :text_to_display, :link
    # attribute mapping from ruby-style variable name to JSON key
    def self.attribute_map
      {
        
        # 
        :'address' => :'Address',
        
        # 
        :'area' => :'Area',
        
        # 
        :'screen_tip' => :'ScreenTip',
        
        # 
        :'text_to_display' => :'TextToDisplay',
        
        # 
        :'link' => :'link'
        
      }
    end

    # attribute type
    def self.swagger_types
      {
        :'address' => :'String',
        :'area' => :'CellArea',
        :'screen_tip' => :'String',
        :'text_to_display' => :'String',
        :'link' => :'Link'
        
      }
    end

    def initialize(attributes = {})
      return if !attributes.is_a?(Hash) || attributes.empty?

      # convert string to symbol for hash key
      attributes = attributes.inject({}){|memo,(k,v)| memo[k.to_sym] = v; memo}

      
      if attributes[:'Address']
        self.address = attributes[:'Address']
      end
      
      if attributes[:'Area']
        self.area = attributes[:'Area']
      end
      
      if attributes[:'ScreenTip']
        self.screen_tip = attributes[:'ScreenTip']
      end
      
      if attributes[:'TextToDisplay']
        self.text_to_display = attributes[:'TextToDisplay']
      end
      
      if attributes[:'link']
        self.link = attributes[:'link']
      end
      
    end

  end
end

File: SDKs/Aspose.Cells_Cloud_SDK_For_.NET/src/Com/Aspose/cells/Model/Hyperlink.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace Com.Aspose.Cells.Model {
  public class Hyperlink {
    public string Address { get; set; }

    public CellArea Area { get; set; }

    public string ScreenTip { get; set; }

    public string TextToDisplay { get; set; }

    public Link link { get; set; }

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class Hyperlink {\n");
      sb.Append("  Address: ").Append(Address).Append("\n");
      sb.Append("  Area: ").Append(Area).Append("\n");
      sb.Append("  ScreenTip: ").Append(ScreenTip).Append("\n");
      sb.Append("  TextToDisplay: ").Append(TextToDisplay).Append("\n");
      sb.Append("  link: ").Append(link).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  }
